src/app/common/search-input/search-input.component.html

```diff
@@ -1,5 +1,15 @@
 <div class="search-input-container d-flex">
-	<input [placeholder]="placeholder" type="text" class="form-control" [(ngModel)]="search" (keyup)="onKeyup()">
+	<input
+		type="text"
+		[placeholder]="placeholder"
+		class="form-control"
+		[(ngModel)]="search"
+		(keyup)="onKeyup()"
+		(input)="onInput()"
+	/>
 	<span *ngIf="search.length === 0" class="icon fa fa-search"></span>
 	<span *ngIf="search.length > 0" (click)="clearSearch($event)" class="icon fa fa-times"></span>
 </div>
+<div *ngIf="showMinCountMessage" [@minCount] class="min-count">
```

Comment:
Could we also have an option not show the min count message?  I did a quick test enabling it on the admin pages and it is causing the size of nearby buttons to change.

![Kapture 2020-10-23 at 11 09 19](https://user-images.githubusercontent.com/23661980/97021632-137d1280-1521-11eb-9ba0-5f32b419bd5f.gif)

Reply:
@jrassa do you mean that you want to still enforce the minimum character count before emitting the search value, but simply not show the message?